bin/safe-regex2.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+'use strict'
+
+const { resolve } = require('node:path')
+const { readFileSync } = require('node:fs')
+const { parseArgs } = require('node:util')
+const { safeRegex } = require('../index.js')
+
+const { version } = JSON.parse(
+  readFileSync(resolve(__dirname, '..', 'package.json'), 'utf8')
+)
```

Comment:
```suggestion
const { version } = require('../package.json')
```

This would also work?